common/components/ui/Identicon.tsx

```diff
@@ -11,7 +11,6 @@ interface Props {
 export default function Identicon(props: Props) {
   const size = props.size || '4rem';
   const { address, className = '' } = props;
-  // FIXME breaks on failed checksums
```

Comment:
This comment isn't true, it was only on addresses with no hex prefix (which happens incredibly infrequently) so I felt it wasn't very actionable.